### PR TITLE
Create karbon.yaml

### PR DESCRIPTION
**Pull request by tuxtof**

### CODE REVIEW

**Comment by tuxtof** [comment]
CLA signing seems OK

**Comment by tuxtof** [comment]
normally all is OK

/assign @corneliusweig

File: plugins/karbon.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/nutanix/kubectl-karbon/releases/download/v0.2.0/kubectl-karbon_v0.2.0_windows_amd64.tar.gz
+    sha256: 8d8485d868e8f604e43c5926523ef77cccab11688daff1c5ac079a3079c8e8e4
+    bin: kubectl-karbon
+  shortDescription: Connect to Nutanix Karbon cluster
+  description: |
+    Usage:
```
Comment (ahmetb) [inline plugins/karbon.yaml:46]: No need to add usage here if you already support "kubectl karbon" to show a help message.

**Review by ahmetb — CHANGES_REQUESTED** [review]
Mostly lgtm. 
Do you import client-go auth plugins in your code?

File: plugins/karbon.yaml (hunk 1 above)
Comment (tuxtof) [reply plugins/karbon.yaml:46]: no don't use  client-go auth plugins, not necessary

ok i will remove usage

Comment (tuxtof) [reply plugins/karbon.yaml]: done @ahmetb

File: plugins/karbon.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    uri: https://github.com/nutanix/kubectl-karbon/releases/download/v0.2.0/kubectl-karbon_v0.2.0_linux_arm64.tar.gz
+    sha256: 64e522186956b17ab7ae6e05217b3c7055a3feb52c18f1d394110d0608033242
+    bin: kubectl-karbon
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/nutanix/kubectl-karbon/releases/download/v0.2.0/kubectl-karbon_v0.2.0_windows_amd64.tar.gz
+    sha256: 8d8485d868e8f604e43c5926523ef77cccab11688daff1c5ac079a3079c8e8e4
+    bin: kubectl-karbon
```
Comment (ahmetb) [inline plugins/karbon.yaml:43]: I believe this needs to be an .exe?

You might wanna try KREW_OS=windows and see how it gets symlinked (it needs an exe suffix i believe)

**Review by ahmetb — COMMENTED** [review]
Can be fixed later.

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!


Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Comment by ahmetb** [comment]
well actually it seems the .exe is causing the test to fail so you actually need to fix it 😇

**Comment by tuxtof** [comment]
normally problem is fixed, bad copy and paste for the windows selector

**Comment by tuxtof** [comment]
thanks @ahmetb

**Pull request merged**